chromadb/config.py

```diff
@@ -124,7 +124,7 @@ def empty_str_to_none(cls, v: str) -> Optional[str]:
     chroma_server_ssl_enabled: Optional[bool] = False
 
     chroma_server_ssl_verify: Optional[Union[bool, str]] = None
-    chroma_server_api_default_path: Optional[str] = "/api/v1"
+    chroma_server_api_default_path: Optional[str] = "/api/v2"
```

Comment:
Should we make this a version enum instead of a path? This can lead to configurations we don't want to exist